main.py

```diff
@@ -77,13 +77,18 @@ def login_user():
     username = content['username']
     password = content['password']
 
-    key = client.key("users", username)
-    user = client.get(key)
-    if not user or not verify_password(user['password'], password):
+    query = client.query(kind="users")
+    query.add_filter("username", "=", username)
+    user = list(query.fetch())
+
+    fUser = user[0]
+    user_id = fUser.id
+
+    if not user or not verify_password(user[0]['password'], password):
         return jsonify(ERROR_401), 401
 
     token = jwt.encode({
-        'public_id': username,
+        'public_id': user_id,

```

Comment:
can you also add the username in the token as well? `'username': username`.

Reply:
Yes it should be added now